app/models/patient.rb

```diff
@@ -492,6 +492,72 @@ def calc_current_age
     age
   end
 
+  # Returns a representative FHIR::Patient for an instance of a Sara Alert Patient. Uses US Core
+  # extensions for sex, race, and ethnicity.
+  # https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-patient.html
+  def as_fhir
+    FHIR::Patient.new(
+      meta: FHIR::Meta.new(lastUpdated: updated_at.strftime('%FT%T%:z')),
+      id: id,
+      name: [FHIR::HumanName.new(given: [first_name, middle_name].reject(&:blank?), family: last_name)],
+      telecom: [
```

Comment:
Other possible fields we can consider mapping to:
Active <- Patient.monitoring
Telecom.system <- Primary contact method
Telecom.period <- preferred contact time
Contact <- Link to responder

Reply:
I like it! Will add.